packages/css-size/src/index.js

```diff
@@ -1,10 +1,24 @@
+import zlib from 'zlib';
 import prettyBytes from 'pretty-bytes';
 import postcss from 'postcss';
-import { sync as gzip } from 'gzip-size';
-import { sync as brotli } from 'brotli-size';
 import Table from 'cli-table3';
 import nano from 'cssnano';
 
+/* Returns the size of the gzipped input */
+function gzip(input) {
+  return zlib.gzipSync(input, { level: 9 }).byteLength;
+}
+
+/* Returns the size of the brotli-compressed input */
+function brotli(input) {
+  return zlib.brotliCompressSync(input, {
```

Comment:
https://github.com/erwinmombay/brotli-size/blob/6247b27b4c9c635cce5a6a781248fa0196208426/src/index.ts#L47
The original code first converts the string to a buffer, this is so it can then get the `byteSize` of the input to use as a hint, we can just call `Buffer.byteLength()` from our existing `getBinarySize()` utility.